models/nnlo/NNLO_bbbar_pp_y05.txt

```diff
@@ -0,0 +1,13 @@
+NNLO predictions from arXiv:2010.11906
```

Comment:
do you think that it is useful to keep this file in the repo for the analyses, even if it will likely not be used for any analysis? If it is for bookkeeping, an alternative solution might be to push it in this other repo https://github.com/DmesonAnalysers/Models

Reply:
I can move this file to the other repo